components/Bottom.js

```diff
@@ -0,0 +1,138 @@
+import styled from "styled-components";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+export default function Footer() {
+  const { pathname } = useRouter();
+
+  return (
+    <StyledNavigation>
```

Comment:
If this is a footer, as the function name suggests, then i would consider wrapping the returned elements in a [`footer`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer) tag. 

I would also consider renaming the file from `Bottom.js` to `Footer.js`